ui/overview_page.go

```diff
@@ -748,10 +842,10 @@ func layoutBalance(gtx layout.Context, amount string, c pageCommon) layout.Dimen
 	mainText, subText := breakBalance(amount)
 	return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Baseline}.Layout(gtx,
 		layout.Rigid(func(gtx C) D {
-			return c.theme.H6(mainText).Layout(gtx)
+			return c.theme.Label(values.MarginPadding20, mainText).Layout(gtx)
```

Comment:
Kindly define an equivalent of this dimension as TextSize. Text size should be defined as "sp" units rather than "dp"